libcontainerd/remote_unix.go

```diff
@@ -429,6 +436,9 @@ func (r *remote) runContainerdDaemon() error {
 }
 
 func setOOMScore(pid, score int) error {
+	if goruntime.GOOS == "solaris" {
```

Comment:
nit: I usually prefer having a dedicated file with a stub than checking `GOOS`. Easier to maintain and expand IMHO.